pkg/operator/apiserver/audit/audit_policies_test.go

```diff
@@ -155,6 +155,32 @@ func TestNewAuditPolicyPathGetter(t *testing.T) {
 	}
 }
 
+func TestDefaultPolicy(t *testing.T) {
+	scenarios := []struct {
+		name string
+	}{
+		{
+			name: "Get default audit policy for the kube-apiserver",
+		},
```

Comment:
we don't need the following test case, it has the same setup as ^

Reply:
removed